en/apispec/openapi.json

```diff
@@ -3057,311 +2530,3053 @@
+                      "properties": {
+                        "id": {
+                          "type": "string",
+                          "description": "ID of a specific HDT billing item."
+                        }
+                      }
+                    }
+                  }
+                }
+              },
+              "examples": {
+                "example-1": {
+                  "value": {
+                    "billingItems": [
+                      {
+                        "id": "domains-base"
+                      },
+                      {
+                        "id": "service-regions"
+                      },
+                      {
+                        "id": "ports-80-443"
+                      },
+                      {
+                        "id": "ports-non-80-443"
+                      },
+                      {
+                        "id": "traffic-gb"
+                      },
+                      {
+                        "id": "vpl-bandwidth-mbps"
+                      },
+                      {
+                        "id": "dedicated-ip-domains"
+                      },
+                      {
+                        "id": "raw-log-download-mb"
+                      }
+                    ]
+                  }
+                }
+              }
+            },
+            "application/xml": {
+              "schema": {
+                "type": "object",
+                "properties": {}
+              }
+            }
+          },
+          "description": ""
+        },
+        "tags": [
+          "Reports"
+        ]
+      }
+    },
+    "/report/domains": {
+      "get": {
+        "summary": "Query Domains",
+        "tags": [
+          "Reports"
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "object",
+                  "properties": {
+                    "domains-report": {
+                      "type": "object",
+                      "properties": {
+                        "domains-summary": {
+                          "type": "integer",
+                          "description": "The total count of access domains."
+                        },
+                        "domains-dedicated-ip-service-summary": {
+                          "type": "integer",
+                          "description": "The total count of access domains which enable dedicated IP services."
+                        },
+                        "domains": {
+                          "type": "string",
+                          "description": "List the hdt domains."
+                        },
+                        "domains-dedicated-ip-service": {
+                          "type": "string",
+                          "description": "List the hdt domains which enabled dedicated IP services."
```

Comment:
Capitalize "hdt"
I recommend rewriting as "List of HDT domains which have enabled dedicated IP services"